content/posts/2018-11-05_naming-sass-color-variables.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Naming Sass color variables
+subTitle: My approach to name color variables
+tags: ["sass", "styles", "colors", "naming", "variables"]
+cover: /img/sass.svg
+postAuthor: Krzysztof Grziwok
+---
+
+# Introduction
+
+Sass is great tool, besides many great features, it gives us ability to declare a custom variables and use it across the project.
+One of the basic use of variables in stylesheets are the colors in our application.
+There are many ways to implement this, however not all stand the test of time.
+In this short post I will show you my personal approach which I used in a few projects and it worked pretty well.
+
+# Problem
+
+When we start a new project, we tend to start small and we usually forget about scalability of our variables or maps.
+The most common use case which I spot in projects, is that we name colors as they are represented visually by the name of primary color.
+
+```css
+$blue: #2B42E9;
+$grey: #D6D7E1;
+$dark: #010105;
+```
+
+Above implementation is fine, but only in projects where style guide contains only a few variations of colors and style guide is defined since the beginning.
```

Comment:
```suggestion
Above implementation is fine, but only in projects where the style guide contains only a few variations of colors and style guide is defined since the beginning.
```